ext/standard/tests/array/array_merge_recursive_error.phpt

```diff
@@ -9,19 +9,8 @@ Test array_merge_recursive() function : error conditions
 
 echo "*** Testing array_merge_recursive() : error conditions ***\n";
 
-// Zero arguments
```

Comment:
Without this code this file doesn't do anything useful anymore -- you can delete it entirely.

Reply:
I was not sure what to do. It's gone now.